proto/store/store/user.proto

```diff
@@ -15,6 +15,17 @@ enum PrincipalType {
   SERVICE_ACCOUNT = 2;
   // SYSTEM_BOT represents the internal system bot performing operations.
   SYSTEM_BOT = 3;
+  // WORKLOAD_IDENTITY represents external CI/CD workload identity.
+  WORKLOAD_IDENTITY = 4;
+}
+
+// ProviderType identifies the CI/CD platform for workload identity.
+enum ProviderType {
```

Comment:
Move it inside WorkloadIdentityConfig and just call it GITHUB without "PROVIDER_" prefix.
And can we just keep GITHUB only for now?